### PR TITLE
Feature-main-proyect

### PR DESCRIPTION
**Pull request by cvalencia1991**
create the second branch to a pull request

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test.md
```
Comment (vechicin) [inline .gitignore:1]: The `test.md` file is not being ignored. The reason for that is likely that you pushed the file before adding it to the `.gitignore` file. Please research how to use the `git rm --cached` command in order to fix this issue.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @cvalencia1991 ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job setting up linters.

## Required Changes ♻️

- [ ] Your PR title and description are not descriptive. Please change them to something descriptive of what is found in the PR. **DO NOT OPEN A NEW PR WHEN DOING THIS**.
- [ ] Your branch name is not descriptive of a feature or action either. Please change it.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (cvalencia1991) [reply .gitignore]: good evening i change the name of the branch and the name of the PR , can you see it please

**Review by MahmoudBakr23 — COMMENTED** [review]
Hi @cvalencia1991  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cvalencia1991** [comment]
thank you so much, but in the dashboard appear submit can you tellme why please

**Pull request closed without merging**